roles/openshift_router/vars/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+openshift_cert_parent_dir: /var/lib/openshift
+openshift_cert_relative_dir: openshift.local.certificates
+openshift_cert_dir: "{{ openshift_cert_parent_dir }}/{{ openshift_cert_relative_dir }}"
+openshift_config: "{{ openshift_cert_dir }}/admin/.kubeconfig"
```

Comment:
Since this should probably run on the master, we should probably just use the configured ~/.config/openshift/.config credentials